.github/workflows/static_checks.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           if [ "${{ github.event_name }}" == "pull_request" ]; then
-            files=$(gh pr diff ${{ github.event.pull_request.number }} --name-only)
```

Comment:
I have no idea where GH gets this from, both mine and upstream `master` branches are up to date...

Reply:
Oh wait

Reply:
God dammit

Reply:
One sec 🙄

Reply:
You broke your branch by not rebasing it again

Reply:
There we go 🎊

Reply:
> You broke your branch by not rebasing it again

The problem was that I didn't sync my local `master` branch with origin and I got tunnel-visioned.